docs/segments/manage_segments.rst

```diff
@@ -3,4 +3,493 @@
+* ``birthday -7 days`` / ``anniversary -7 days``
+* ``today`` / ``tomorrow`` / ``yesterday``
+* ``this week`` / ``last week`` / ``next week``
+* ``this month`` / ``last month`` / ``next month``
+* ``this year`` / ``last year`` / ``next year``
+
+Example (Consider that today is ``2022-03-05``):
+
+* ``Date identified equals last week`` returns all Contacts identified between 2022-03-01 and 2022-03-07 (Monday - Sunday).
+* ``Date identified less than last week`` returns all Contacts identified before 2022-02-22.
+* ``Date identified equals last month`` returns all Contacts identified between 2022-02-01 and 2022-02-28.
+* ``Date identified greater or equal last year`` returns all Contacts identified 2021-01-01 and after.
+* ``Date identified greater than last year`` returns all Contacts identified after 2021-12-31.
+* ``Custom Contact date field equal birthday -1 day`` returns all Contacts identified every year on 03-04 (4th march).
+* ``Custom Contact date field equal anniversary -1 month`` returns all Contacts identified every year on 02-04 (4th february)
+
+Once you have created your Segment, any applicable Contact automatically add through the execution of a cron job. This is the essence of Segments.
+
+To keep the Segments current, create a cron job that executes the following command at the desired interval:
+
+``php /path/to/mautic/bin/console mautic:Segments:update``
+
+Through the execution of that command, it adds Contacts that match the filters and removes Contacts that no longer match. Any Contacts that were manually added will remain part of the list regardless of filters.
+
+.. vale off
+
+Delete all Contacts in a Segment
+********************************
+
+.. vale on
+
+Filter the Contacts in the Segment. The batch delete action in the Contact table allows deletion of up to 100 Contacts at one time. 
+This is a performance precaution since deleting more Contacts at one time could cause issues. Hundreds of Contacts can use this feature.
+
+1. Select the checkboxes next to those Contacts.
+
+2. Click the green arrow which appears at the top of the column.
+
+3. Select **Delete Selected** from the list.
+
+4. Choose the **Contacts** you’d like to delete in a Segment.
+
+5. Click **Delete**.
+
+.. image:: images/select-delete.png
+    :width: 200
+    :alt: Screenshot showing Deleting all Contacts in a Segment.
+
+But deleting thousands of Contacts this way in one Segment becomes a tedious task. Luckily, there is a trick how to let the background workers do the job for you.
+
+1. Create a simple Campaign which has the Segment as the source
+
+2. Use the :ref:`Delete contact action<using the campaign builder>`.
+
+This way the ``mautic:campaign:update`` and ``mautic:campaign:trigger`` commands deletes all the Contacts in the Segment. As well as all the Contacts which adds to the Segment in the future. 
+It's all done automatically in the background.
+It's necessary to configure the cron jobs. Deleted Contacts can't be recovered.
```

Comment:
Link to the relevant section of the cron jobs documentation.

Personally I would put the part about the contacts not being recoverable in a big red warning box and advise folks to use this feature very carefully!